generate_outputs.py

```diff
@@ -142,10 +126,25 @@
         log.debug(f"Loaded {len(messages)} messages")
         follow_up_survey_datasets.append(messages)
```

Comment:
Don't you need to coalesce here as well?

(I thought the surveys were like demog flows, in that people could only answer each question once, but might do so over multiple runs. If this is the case, do you want to separate into a new row per run, or have just one row containing all of the follow-up data?)